htdocs/projet/tasks/time.php

```diff
@@ -207,6 +207,9 @@
 					$error++;
 				}
 			}
+			$page = DOL_URL_ROOT . '/projet/tasks/time.php?withproject=1&projectid='.$projectid;
```

Comment:
Same: redirect must be done only if create was a success. Other case shoul dbe alredy managed by having the $action set to the action to redo (for example here $action = 'createtime'; line 186)

Reply:
let me review the PR once over before you comment on all steps